@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -568,40 +620,39 @@ export class SolidLineMaterial extends THREE.RawShaderMaterial
 
     /**
      * Size of the gaps between dashed segments.
+     *
+     * @note You may also need to set [[dashSize]].
+     * @see dashSize.
      */
     get gapSize(): number {
         return this.uniforms.gapSize.value as number;
     }
     set gapSize(value: number) {
         this.uniforms.gapSize.value = value;
-        this.defines.DASHED_LINE = this.gapSize > 0.0 ? 1 : 0;
+        setShaderMaterialDefine(this, "DASHED_LINE", value > 0.0 ? 1 : 0);
```

Comment:
Use "boolean" define (See previous comment).

Reply:
As before.... old code ... more cleaning required - not exactly related to my change, but could be done in some follow up.